### ISSUE TITLE
Auto indentation in Stitch

### ISSUE THREAD

**Issue by juzi** [bscotch/stitch#252]
I really like your plugin, but when I try to auto-indent a region, vscode tells me that there is no formatter installed.
Is there a way to get auto-indentation? Or do you do this externally when saving/committing a file?
This is one of the features that I miss in GML2 and it would be the number one reason to switch to vscode.

**Comment by adam-coster** [comment]
You're correct that Stitch doesn't include a formatter. I don't do much game coding, but if I did then that'd also be a high priority for me, since I really can't stand manual formatting! Our main programmer doesn't want a formatter, though, so it's been a low priority item.

I was thinking about creating an extension with [Gobo](https://github.com/Pizzaandy/Gobo) so people could use that along with Stitch, but some of its formatting opinions aren't compatible enough with how our main game programmer wants things formatted so I haven't put the time in.
